src/main/java/org/jenkinci/plugins/mock_slave/MockCloud.java

```diff
@@ -63,12 +66,26 @@
     public final Node.Mode mode;
     public final int numExecutors;
     public final String labelString;
+    /** only null when deserializing old configs */
+    private @Nullable Boolean oneShot;
```

Comment:
I do _not_ want `@CheckForNull` because most usages do not, in fact, need to check for null.